packages/crypto-wallet-core/src/derivation/paths.ts

```diff
@@ -27,6 +27,11 @@ export const Paths = {
     livenet: "m/44'/2'/",
     testnet: "m/44'/2'/"
   },
+  MATIC: {
+    mainnet: "m/44'/60'/", // actual matic derivation path is 966 but users will expect address to be same as ETH
```

Comment:
Maybe we should say "the official matic ..." instead of "actual". "Actual" makes it sound like `m/44'/60'/` will somehow be translated into `m/44'/966'/`

Reply:
Fixed in my latest commit.